classes/class-wc-connect-shipping-label.php

```diff
@@ -362,6 +364,14 @@ protected function get_states_map() {
 		}
 
 		public function should_show_meta_box() {
+			if ( null === $this->show_metabox ) {
```

Comment:
The idea is to avoid running all the code in `should_show_metabox()`. The value is needed twice: Once to decide if the pointer is being shown, and once to decide if the metabox is to be show.

I don't think two instances of `WC_Connect_Shipping_Label`, I'm passing the same one to the constructor of `_NUX` that's used elsewhere...